xml/api/resources-2.4.rng

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is it confusing to have xml/resources-* and xml/api/resources-*?

Reply:
I don't think so. For 2.1.0 I'm thinking of moving xml/*{rng,xsl} to xml/cib/ anyway.